### DISCUSSION TITLE
How do I use the Code Llama
Model?

### DISCUSSION THREAD

**Discussion by michabbb** [louisgv/local.ai#106] [Q&A]
after downloaded, i have files like this:

consolidated.00.pth
consolidated.01.pth
consolidated.02.pth
.....

do I have to convert them to a *.bin file ? if yes, how ? ;-)

thanks !!

**Comment by louisgv** [comment]
We would need to find a version of Code Llama that is converted to GGML - also will need to update the build as well :d...

**Reply by BLamy** [reply]
https://huggingface.co/TheBloke/CodeLlama-34B-GGML
